src/lib/components/explorer/genome-filter/FilterType.svelte

```diff
@@ -9,22 +10,26 @@
 </script>
 
 <div class="flex flex-row justify-center justify-items-center gap-16 {$$props.class ?? ''}">
-  <CardButton
-    data-testid="gene-variant-option"
-    title="Variants by gene name"
-    subtitle="Ex: Rare BRCA1 variants with high severity"
-    size="other"
-    class="w-1/4 h-20 min-h-20 variant-ringed-primary"
-    active={active === Option.Genomic}
-    on:click={() => dispatch('select', { option: Option.Genomic })}
-  />
-  <CardButton
-    data-testid="snp-option"
-    title="Specific SNPs"
-    subtitle="Ex: chr17,35269878,G,A"
-    size="other"
-    class="w-1/4 h-20 min-h-20 variant-ringed-primary"
-    active={active === Option.SNP}
-    on:click={() => dispatch('select', { option: Option.SNP })}
-  />
+  {#if features.explorer.enableGENEQuery}
```

Comment:
I think we should just not show the buttons at all if only one option is enabled.
@emhug Correct if I am wrong

Reply:
that's why I hide the button that leads to this page when that condition is met

Reply:
Is there some place else I should disable it too? Like should I make this page 404 somehow?